src/components/BulletChart.tsx

```diff
@@ -0,0 +1,65 @@
+// (C) 2007-2020 GoodData Corporation
```

Comment:
This file is derived from the bar chart, just the stack by was removed and measures converted from an array to single value prop